### ISSUE TITLE
beta.7 release

### ISSUE THREAD

**Issue by ChristopherBiscardi** [FezVrasta/react-popper#182]
I'm running into the infinite loop issue (#140) fixed by #166. I have a workaround from the thread in #140 working (so not time sensitive to the point where I would vendor react-popper), but I'd like to upgrade to the "official" solution at some point. It seems like master can be cut as beta.7 (build is passing) so can there be a release when you have a moment?

**Comment by giladgray** [comment]
@FezVrasta any update here?

**Comment by ChristopherBiscardi** [comment]
yeah, that seems like it could be nicer (is the only benefit not using the additional class properties?), but tbh I already wrote/merged the code and don't plan on opening a PR to refactor it until beta.7 is out (Just so much to do lol).

We also use react-popper as the underlying infrastructure in our design system popovers and tooltips so I end up with 3 refs to handle in my `<Reference>`s (one for react-popper, one for us to check `.contains`, and one optional for the popover consumer that possibly comes in on `children`). Ends up looking something like this:

```js
getRef = clickTargetRef => {
  this.ref(clickTargetRef);
  this.childRef && this.childRef(clickTargetRef);
  this.clickTarget = clickTargetRef;
};
...
<Reference>
{
  ({ ref }) => {
    this.ref = ref;
    this.childRef = children.ref;
    return React.cloneElement(children, {
      [typeof children.type === "string" ? "ref" : "innerRef"]: this.getRef,
      onClick: this.onClickTargetClick
    });
  };
}
</Reference>
```

**Comment by piecyk** [comment]
yeah 👍no refactor needed :) Just saying that basic there will be no difference with beta.7 innerRef vs memoize setter of ref

**Comment by FezVrasta** [comment]
Published 1.0.0
